### ISSUE TITLE
Support for DockerFile for fly.io deployment

### ISSUE THREAD

**Issue by rhonivanpareja** [wtfnotavailable/Discord-MusicBot#209]
Any change to have Dockerfile to be able to deploy on fly.io? thanks!

**Comment by SIGTERM-015** [comment]
I don't think you can deploy it on fly.io, but here is the Dockerfile for the bot 
https://github.com/wtfnotavailable/Discord-MusicBot/blob/develop/docker/djs-bot/Dockerfile

**Comment by DarrenOfficial** [comment]
I don't think it's possible, even if it is; theres no benefits of using fly.io/deploying near user. Since the audio quality is affected by lavalink itself, and cold start is an issue for this bot. (It's not designed to be spun up / down in a matter of ms)
